tests/parm/model_configure_regional_rrfs_a.IN

```diff
@@ -0,0 +1,102 @@
+total_member:            1
+PE_MEMBER01:             496
+start_year:              2021
+start_month:             5
+start_day:               31
+start_hour:              3
+start_minute:            0
+start_second:            0
+nhours_fcst:             @[FHMAX]
+RUN_CONTINUE:            .false.
+ENS_SPS:                 .false.
+dt_atmos:                60
+cpl:                     .false.
+calendar:                'julian'
+memuse_verbose:          .false.
+atmos_nthreads:          2
+use_hyper_thread:        .false.
+ncores_per_node:         40
+debug_affinity:          .true.
+restart_interval:        @[RESTART_INTERVAL]
+output_1st_tstep_rst:    .false.
+print_esmf:              .false.
```

Comment:
`total_member`, `PE_MEMBER01`, `RUN_CONTINUE`, `ENS_SPS`, `memuse_verbose`, `atmos_nthreads`, `use_hyper_thread`, `ncores_per_node`, `debug_affinity`, `print_esmf` are not used (anymore) by the model.

Reply:
removed